load_sample_data.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python
+import json
+import re
+import subprocess
+import sys
+import time
+import uuid
+from dataclasses import dataclass
+from typing import Literal
+
+from decouple import config
+from python_on_whales import DockerException, docker
+
+
+WEB_SERVICE_NAME = config("WEB_SERVICE_NAME", default="web")
+CACHE_SERVICE_NAME = config("CACHE_SERVICE_NAME", default="cache")
+UPSTREAM_DB_SERVICE_NAME = config("UPSTREAM_DB_SERVICE_NAME", default="upstream_db")
+DB_SERVICE_NAME = config("DB_SERVICE_NAME", default="db")
+
+MEDIA_TYPES = ["image", "audio"]
+MediaType = Literal["image", "audio"]
+
+
+##############
+# Subscripts #
+##############
+
+
+PSQL = "psql -U deploy -d openledger -v ON_ERROR_STOP=1 -X"
+DJ_SHELL = "python manage.py shell"
+
+
+##########
+# Models #
+##########
+
+
+@dataclass
+class Provider:
+    identifier: str
+    name: str
+    url: str
+    media_type: str
+    filter_content: bool = False
+
+    @property
+    def sql_value(self) -> str:
+        fields = ", ".join(
+            [
+                "now()",
+                f"'{self.identifier}'",
+                f"'{self.name}'",
+                f"'{self.url}'",
+                f"'{self.media_type}'",
+                str(self.filter_content).lower(),
+            ]
+        )
+        return f"({fields})"
+
+
+@dataclass
+class Column:
+    name: str
+    type: str
+
+
+###########
+# Helpers #
+###########
+
+
+def compose_exec(service: str, bash_input: str) -> str:
+    """
+    Run the given input inside a Bash shell inside the container.
+
+    :param service: the name of the service inside which to execute the commands
+    :param bash_input: the input for the Bash shell
+    :return: the output of the operation
+    """
+
+    bash_input = re.sub(r"\n\s{8}", r"\n", bash_input)
+    return docker.compose.execute(service, ["/bin/bash", "-c", bash_input], tty=False)
+
+
+def copy_table_upstream(
+    name: str, target_name: str = None, delete_if_exists: bool = True
+):
+    """
+    Copy the given table from the downstream DB to the upstream DB. Any existing table
+    with the same name can be deleted before copying and the table can be renamed after
+    copying.
+
+    :param name: the name of the source table to copy
+    :param target_name: the name to assign to the copied table
+    :param delete_if_exists: whether to delete any existing tables with the target name
+    """
+
+    if target_name is None:
+        target_name = name
```

Comment:
```suggestion
    target_name = target_name or name
```

Reply:
Would this work, or is there a case where valid target_name can be falsy?

Reply:
That would definitely work. I just went with `is None` because that seemed to more clearly convey the intent.